src/components/Table/RowState.ts

```diff
@@ -161,6 +177,30 @@ export class RowState {
       localStorage.setItem(this.persistCollapse, JSON.stringify(collapsedIds));
     }
   }
+
+  private getVisibleImmediateChildrenSelectedStates(
```

Comment:
Fwiw I'd be fine with `getVisibleChildrenStates` just to nudge it a little shorter